ai-safety-debate/debate/debate.py

```diff
@@ -1,7 +1,9 @@
 import numpy as np
 from .debate_state import DebateState
-from util.visualization import plot_image_mask
-
+try:
+    from util.visualization import plot_image_mask
+except ModuleNotFoundError:
+    visualization = False
```

Comment:
visualization is nor defined if there is not exception

Reply:
Calling it `visualization_available` would be more clear

Reply:
it would also be longer